### PR TITLE
Video orientation

### PR DESCRIPTION
**Pull request by adamstyrc**

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -25,4 +25,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ebd43b443038e611b86ede96e613bd6033c49497
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.10.0.rc.1
```
Comment (Sauraus) [inline example/ios/Podfile.lock]: Lets not use rc.1 please

**Review by Sauraus — CHANGES_REQUESTED** [review]

File: ios/Classes/FileFetcher.swift [hunk 2]
```diff
@@ -173,13 +178,39 @@ class FileFetcher {
                 path: url,
                 thumbnailPath: cachePath?.path,
                 orientation: 0,
```
Comment (jlalvarez18) [inline ios/Classes/FileFetcher.swift]: Should orientation be 0? Or `orientation` defined above?

File: android/src/main/kotlin/com/kasem/media_picker_builder/MediaFile.kt [hunk 3]
```diff
@@ -14,6 +14,7 @@ class MediaFile(
          * Only degrees 0, 90, 180, 270 will work.
          * */
         var orientation: Int,
+        var videoOrientation: Int,
```
Comment (jlalvarez18) [inline android/src/main/kotlin/com/kasem/media_picker_builder/MediaFile.kt]: This seems confusing that there is already an `orientation` property on MediaFile on android and ios. Why not leverage what is already there?

**Review by jlalvarez18 — CHANGES_REQUESTED** [review]

File: android/src/main/kotlin/com/kasem/media_picker_builder/MediaFile.kt (hunk 3 above)
Comment (adamstyrc) [reply android/src/main/kotlin/com/kasem/media_picker_builder/MediaFile.kt]: Was weird to me too, but for videos, we use it like for orientation of a thumbnail so I was forced to add another field

Comment (adamstyrc) [reply android/src/main/kotlin/com/kasem/media_picker_builder/MediaFile.kt]: Ok, I'll make sure it's one orientation, working for example and vid_camera app 🙃

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -365,7 +366,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
-				DEVELOPMENT_TEAM = S8QB4VV633;
+				DEVELOPMENT_TEAM = 75LT84MA2Q;
```
Comment (Sauraus) [inline example/ios/Runner.xcodeproj/project.pbxproj:369]: Why did you commit this?

File: example/lib/picker/picker_widget.dart [hunk 5]
```diff
@@ -33,7 +33,8 @@ class PickerWidgetState extends State<PickerWidget> {
   void initState() {
     super.initState();
     MediaPickerBuilder.getAlbums(
-      withImages: widget.withImages,
+      // withImages: widget.withImages,
+      withImages: false,
```
Comment (Sauraus) [inline example/lib/picker/picker_widget.dart:37]: What does this do in the example?

**Pull request merged**